packages/@sanity/schema/src/sanity/validation/types/block.ts

```diff
@@ -102,6 +108,48 @@ function validateMarks(marks, visitorContext, problems) {
   return {...marks, decorators, annotations}
 }
 
+function validateLists(lists, visitorContext, problems) {
```

Comment:
Does this have a unit test?

Reply:
I can't find any existing tests for these validations. We should have them though. But I guess that's outside of the scope here right now.